acceptance-tests/features/register.feature

```diff
@@ -37,6 +40,16 @@ Feature: Users can sign up to the self-service experience
         | "test-user@yahoo.co.uk"                                          |
         | "the-config-file-is-broken-add-a-dot-before-gov.uk@sneakygov.uk" |
 
+  Rule: User validating the links in registration page
+    Scenario Outline:User validating links in enter-email address page
+      Given the user is on the "/register" page
+      When they click on the '<linkName>' link
+      Then they should be directed to the URL '<address>'
+      Examples:
+        | linkName       | address                                                 |
+        | Contact us     | https://www.sign-in.service.gov.uk/contact-us?adminTool |
```

Comment:
This test should be domain independent or domain dynamic.
Verify that link ends in "/contact-us" or make https://www.sign-in.service.gov.uk dynamic depending on environment.

Reply:
updated in the tests